concepts/conditionals/about.md

```diff
@@ -0,0 +1,91 @@
+# About
+
+## If Expression
+
+`jq`'s conditional expression is `if A then B else C end`.
+
+`if-then-else` is a filter like all jq builtins: it takes an input and produces an output.
+
+If the expression `A` produces a "truthy" value, then the `if` filter outputs the result of passing the input to `B`.
+Otherwise the `if` filter outputs the result of passing the input to `C`.
```

Comment:
The manual says:
> `if A then B else C end` will act the same as `B` if `A` produces a value other than false or null, but act the same as `C` otherwise.

I wasn't happy with "act the same as".

Maybe

> If the expression `A` produces a "truthy" value, then the `if` filter evaluates `B`.
> Otherwise it evaluates `C`.
> The input to the `if` filter will be passed to `B` or `C`.

Reply:
Yup! That sounds much better to me.